examples/resources/grafana_cloud_plugin/resource.tf

```diff
@@ -0,0 +1,5 @@
+resource "grafana_cloud_plugin_installation" "test" {
```

Comment:
The name of the folder has to match the resource name, otherwise it doesn't show up in the docs